lighthouse-treemap/tsconfig.json

```diff
@@ -5,4 +5,7 @@
     "types",
     "../types",
   ],
+  "compilerOptions": {
```

Comment:
is this one step ahead?

Reply:
It's necessary, although I couldn't tell you exactly why. Without it:

```

lighthouse-viewer/app/src/firebase-auth.js:10:8 - error TS1259: Module '"idb-keyval"' can only be default-imported using the 'esModuleInterop' flag

10 import idbKeyval from 'idb-keyval';
          ~~~~~~~~~

  node_modules/idb-keyval/typings.d.ts:20:3
    20   export = idbKeyVal;
         ~~~~~~~~~~~~~~~~~~~
    This module is declared with using 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.

lighthouse-viewer/app/src/firebase-auth.js:80:61 - error TS7006: Parameter '_' implicitly has an 'any' type.

80       return idbKeyval.set('accessToken', accessToken).then(_ => accessToken);
                                                               ~

lighthouse-viewer/app/src/github-api.js:12:8 - error TS1259: Module '"idb-keyval"' can only be default-imported using the 'esModuleInterop' flag

12 import idbKeyval from 'idb-keyval';
          ~~~~~~~~~

  node_modules/idb-keyval/typings.d.ts:20:3
    20   export = idbKeyVal;
         ~~~~~~~~~~~~~~~~~~~
    This module is declared with using 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.

lighthouse-viewer/app/src/github-api.js:152:47 - error TS7006: Parameter '_' implicitly has an 'any' type.

152       return idbKeyval.set(id, response).then(_ => {
                                                  ~

lighthouse-viewer/app/src/github-api.js:154:9 - error TS2578: Unused '@ts-expect-error' directive.

154         // @ts-expect-error - TODO(bckenny): tsc unable to flatten promise chain here
            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 5 errors.

Files:              291
Lines:           130197
Nodes:           506868
Identifiers:     181521
Symbols:         174664
Types:            92690
Instantiations:  118072
Memory used:    295482K
I/O read:         0.04s
I/O write:        0.00s
Parse time:       1.26s
Bind time:        0.58s
Check time:       5.04s
Emit time:        0.00s
Total time:       6.88s
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Reply:
ah these must be in treemap's dependency graph 👍